### PR TITLE
Issue 778

### LINKED ISSUE #778: Create a Country Guessing Game
# Aim

-Create a game with a blank world map, that allows players to add countries to it. With every correct guess, the player earns a point. 

# Details

- 
image

### PR DESCRIPTION
**Pull request by Shubhrima**
# Description

Added a game, in which the player has to guess the name of the countries on the world map. With every correct guess the player earns a point.

## Fixes #778

778

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [ x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ x] My code follows the style guidelines(Clean Code) of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have created a helpful and easy to understand `README.md`
- [x ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x ] My changes generate no new warnings
- [ x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Have a look at the error logs and fix these issues: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/782/checks?check_run_id=2227796735

**Comment by Shubhrima** [comment]
@kaustubhgupta  I am unable to understand the errors. Could you please help me out!

**Comment by Shubhrima** [comment]
@kaustubhgupta I am unable to understand the errors. Could you please help me out! Also, I have used the turtle module here.

File: Guess the Countries/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Package/Script Name
```
Comment (kaustubhgupta) [inline Guess the Countries/README.md:1]: Remove this line

File: Guess the Countries/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Guess the Country
+
+
+<p> Guess the name of the countries on the world map and score a point with every correct guess.</p>
+
+
+Output
```
Comment (kaustubhgupta) [inline Guess the Countries/README.md:8]:
```suggestion
# Output
```

File: Guess the Countries/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<img src='blank.png' width='350'>
```
Comment (kaustubhgupta) [inline Guess the Countries/README.md:9]: Images not rendered:

image

File: Guess the Countries/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+<img src="guessed_countries.png" width='350'>
+
+Author(s)
+Shubhrima Jana
+
```
Comment (kaustubhgupta) [inline Guess the Countries/README.md:14]: Remove these blank lines

File: Guess the Countries/main.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import turtle
+import pandas
+import pylint
+import random
+screen=turtle.Screen()
+
+screen.title('Name the Countries')
+image='world_map.gif'
```
Comment (kaustubhgupta) [inline Guess the Countries/main.py:8]:
```suggestion
image='./Guess the Countries/world_map.gif'
```

File: Guess the Countries/main.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+correct=0
+
+screen.setup(width=800,height=500)
+screen.addshape(image)
+turtle.shape(image)
+
+data=pandas.read_csv("country_names.csv")
```
Comment (kaustubhgupta) [inline Guess the Countries/main.py:15]:
```suggestion
data=pandas.read_csv("./Guess the Countries/country_names.csv")
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
@Shubhrima Your game works. Just do these modifications:

**Comment by Shubhrima** [comment]
I made the necessary changes, @kaustubhgupta kindly review this.

File: Guess the Countries/README.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Guess the Country
+
+<p> Guess the name of the countries on the world map and score a point with every correct guess.</p>
+
+# Output
+<img src="blank.png" width="350"/>
```
Comment (kaustubhgupta) [inline Guess the Countries/README.md:6]:
```suggestion
![blank](blank.png)
```

File: Guess the Countries/README.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+<img src="guessed_countries.png" width="350"/>
```
Comment (kaustubhgupta) [inline Guess the Countries/README.md:7]:
```suggestion

![guessed](guessed_countries.png)
```

File: Guess the Countries/main.py (hunk 5 above)
Comment (kaustubhgupta) [reply Guess the Countries/main.py:8]: This  suggestion is left

File: Guess the Countries/main.py (hunk 6 above)
Comment (kaustubhgupta) [reply Guess the Countries/main.py:15]: This suggestion is left

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Shubhrima** [comment]
I made the necessary changes, @kaustubhgupta kindly review this once again.

**Review by santushtisharma10 — APPROVED** [review]
Nice @Shubhrima 💯

**Comment by santushtisharma10** [comment]
@Kushal997-das

**Comment by santushtisharma10** [comment]
@antrikshmisri @Kushal997-das

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**